Pa saas security xsiam (#19137)
demisto/content: Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/SaasSecurityEventCollector_description.md

> ## Limitations

Reviewer: Also, let's add here how a user should get the client secret and id

Reviewer 2: it already exists in the description file, see below.

> 1) by definition the events that occur lasts only 1 hour in Saas-Security cache, hence if setting the limit to a very low limit while

Reviewer 3: ```suggestion
1) Occurring events expire after one hour in Saas-Security cache, so setting a low limit could cause events to expire if there are a large number of events in the cache.
```

> 1) by definition the events that occur lasts only 1 hour in Saas-Security cache, hence if setting the limit to a very low limit while
>   in the Saas-Security environment there are huge amounts of events, those could be lost in the process of fetching.

Reviewer 3: ```suggestion
1) Occurring events expire after one hour in Saas-Security cache, so setting a low limit could cause events to expire if there are a large number of events in the cache.
```

> 2) the max-fetch/limit parameters to fetch events are limited to be only 100 divisible numbers due to Saas-Security api limitations.

Reviewer 3: ```suggestion
2) The max-fetch/limit parameters to fetch events must be divisible by 100.
```

> 3) The **reset last fetch** has no effect.

Reviewer 3: ```suggestion
3) **reset last fetch** has no effect.
```

> 

Reviewer 3: ```suggestion
4) On initial activation this integration will pull events starting from one hour prior.
5) Calling get_events may take upwards of twenty seconds in some cases.
```

2) The max-fetch/limit parameters to fetch events must be divisible by 100.
3) **reset last fetch** has no effect.
4) On initial activation this integration will pull events starting from one hour prior.
5) Using the ```saas-security-get-events``` command may take upwards of twenty seconds in some cases.
6) In some rare cases more than ```max_fetch``` events could be fetched.

> Creating the Client ID and Client Secret in the SaaS Security UI:

Reviewer 3: Saas

Reviewer 2: i think they only meant to the display on the integration in the yml, no need to change it i guess
